### ISSUE TITLE
Azure CLI - Powershell

### ISSUE THREAD

**Issue by PrHar** [Azure/azure-docs-cli-python#748]
There is not enough "documentation"/"detailed step by step guide" on, what to follow next after downloading MSI installer, in-order to access CLI through Powershell.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8246b90a-efb1-03ce-f642-d3eb2441fc30
* Version Independent ID: 2fdd20ef-7bd3-57bc-7957-93f4bb5377ce
* Content: [Install the Azure CLI for Windows](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?view=azure-cli-latest)
* Content Source: [docs-ref-conceptual/install-azure-cli-windows.md](https://github.com/Azure/azure-docs-cli-python/blob/master/docs-ref-conceptual/install-azure-cli-windows.md)
* Service: **multiple**
* GitHub Login: @sptramer
* Microsoft Alias: **sttramer**

**Comment by Favna** [comment]
1. Install
2. Open Powershell
3. Open API reference, for example [login](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest)
4. run commands as documented

Not sure where you're missing documentation but everything is there.

**Comment by PrHar** [comment]
Hello Favna,

Thank you for following up and sharing the documentation that you mentioned in your link.

I believe, I was updated about the document being updated recently, since I did not see “az login” mentioned anywhere, and I tried figuring out the “next steps to be followed” after installing MSI.

I see now, these details and the link to “Log in with Azure CLI2.0”, which makes the understanding more clear for anyone who visits this documentation for the first time.

-Thanks,
Preeti
